packages/gallery/tests/e2e/styleParams/cubeFitPosition.e2e.spec.js

```diff
@@ -0,0 +1,68 @@
+import GalleryDriver from '../../drivers/pptrDriver';
+import { toMatchImageSnapshot } from '../../drivers/matchers';
+import { GALLERY_CONSTS } from 'pro-gallery-lib';
+
+expect.extend({ toMatchImageSnapshot });
+
+describe('cubeFitPosition - e2e', () => {
+  let driver;
+
+  beforeAll(async () => {
+    driver = new GalleryDriver();
+    await driver.openPage();
+  });
+
+  afterAll(async () => {
+    await driver.closePage();
+  });
+  it('should align to the middle', async () => {
+    await driver.navigate({
+      galleryLayout: GALLERY_CONSTS.layout.GRID,
+      cubeType: GALLERY_CONSTS.cubeType.FIT,
+      cubeFitPosition: GALLERY_CONSTS.cubeFitPosition.MIDDLE,
+    });
+    await driver.waitFor.hookToBeVisible('item-container');
+    const page = await driver.grab.elemScreenshot('.pro-gallery');
+    expect(page).toMatchImageSnapshot();
+  });
+  it('should align to the left', async () => {
+    await driver.navigate({
+      galleryLayout: GALLERY_CONSTS.layout.GRID,
+      cubeType: GALLERY_CONSTS.cubeType.FIT,
+      cubeFitPosition: GALLERY_CONSTS.cubeFitPosition.LEFT,
+    });
+    await driver.waitFor.hookToBeVisible('item-container');
+    const page = await driver.grab.elemScreenshot('.pro-gallery');
+    expect(page).toMatchImageSnapshot();
+  });
+  it('should align to the right', async () => {
+    await driver.navigate({
+      galleryLayout: GALLERY_CONSTS.layout.GRID,
+      cubeType: GALLERY_CONSTS.cubeType.FIT,
+      cubeFitPosition: GALLERY_CONSTS.cubeFitPosition.RIGHT,
+    });
+    await driver.waitFor.hookToBeVisible('item-container');
+    const page = await driver.grab.elemScreenshot('.pro-gallery');
+    expect(page).toMatchImageSnapshot();
+  });
+  it('should align to the top', async () => {
+    await driver.navigate({
+      galleryLayout: GALLERY_CONSTS.layout.GRID,
+      cubeType: GALLERY_CONSTS.cubeType.FIT,
+      cubeFitPosition: GALLERY_CONSTS.cubeFitPosition.TOP,
+    });
+    await driver.waitFor.hookToBeVisible('item-container');
+    const page = await driver.grab.elemScreenshot('.pro-gallery');
+    expect(page).toMatchImageSnapshot();
```

Comment:
@guysopher This entire code is duplicated, makes it impossible to understand the difference between the tests.
Pleaes reuse the duplicated props and the logic parts:
```
    await driver.waitFor.hookToBeVisible('item-container');
    const page = await driver.grab.elemScreenshot('.pro-gallery');
    expect(page).toMatchImageSnapshot();
```